go.mod

```diff
@@ -6,17 +6,16 @@ toolchain go1.22.2
 
 require (
 	code.dny.dev/ssrf v0.2.0
-	github.com/auth0/go-jwt-middleware v1.0.1
+	github.com/auth0/go-jwt-middleware/v2 v2.2.1
 	github.com/avast/retry-go/v4 v4.3.0
 	github.com/bmatcuk/doublestar/v2 v2.0.4
 	github.com/bradleyjkemp/cupaloy/v2 v2.8.0
 	github.com/cenkalti/backoff/v4 v4.3.0
 	github.com/cockroachdb/cockroach-go/v2 v2.3.5
 	github.com/dgraph-io/ristretto v0.1.1
-	github.com/docker/docker v20.10.27+incompatible
+	github.com/docker/docker v27.0.3+incompatible
```

Comment:
This could break dockertest

Reply:
I can also use the same version as dockertest.